ajax/libs/balance-text/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "balance-text",
-  "version": "2.0.0",
-  "filename": "jquery.balancetext.min.js",
-  "description": "A jQuery plugin for implementing balancing of text across lines in a web page",
+  "version": "3.0.0",
```

Comment:
FYI, I updated version in package.json in my repo in [this PR](https://github.com/adobe-webplatform/balance-text/pull/96) but this was not updated. Hopefully, it was due to library name change.